src/smartptrs.h

```diff
@@ -153,6 +154,7 @@ struct ExternalRefCounted {
 namespace Location {
 struct CPU {};
 struct GPU {};
+struct NPU {};
```

Comment:
Is this used anywhere? We've been removing this usage as it's been replaced by the DeviceSpan struct for more specific device usage.

Reply:
This is not used anywhere. We can remove it.

Reply:
Removed in latest push